win32ss/user/user32/windows/message.c

```diff
@@ -2899,8 +2899,7 @@ DWORD
 WINAPI
 RealGetQueueStatus(UINT flags)
 {
-   #define QS_TEMPALLINPUT 255 // ATM, do not support QS_RAWINPUT
-   if (flags & ~(QS_SMRESULT|QS_ALLPOSTMESSAGE|QS_TEMPALLINPUT))
+   if (flags & ~(QS_SMRESULT|QS_ALLPOSTMESSAGE|QS_ALLINPUT))
```

Comment:
`QS_TEMPALLINPUT` hack was added on 7d1736a1a2a39ab506a3fe94fdd0e27aef1d9c46.
Was that wrong? Or what changed since then, that allows to get rid of it?